user_manual/files/access_webgui.rst

```diff
@@ -97,6 +96,17 @@ Overflow Menu
   
 .. figure:: ../images/files_page-4.png
    :alt: Details screen.  
+   
+The **Settings** gear icon at the lower left allows you to show or hide hidden 
+files in your ownCloud Web interface. These are also called dotfiles, because 
+they are prefixed with a dot, e.g. ``.mailfile``. The dot tells your operating 
+system to hide these files in your file browsers, unless you choose to display 
+them. Usually these are configuration files, so having the option to hide them 
+reduces clutter. When you enable or disable showing hidden files, refresh your 
+Web browser to see the change.
```

Comment:
Why refresh the browser? The file list is updated instantly. If not, you might have found a bug ;-) Which browser do you use?
Tested it with Firefox and current core master - file list is updated as expected.